frames/src/main/java/com/checkout/frames/utils/extensions/ViewExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package com.checkout.frames.utils.extensions
+
+import android.graphics.Rect
+import android.view.View
+
+@SuppressWarnings("MagicNumber")
+internal fun View.isKeyboardOpen(): Boolean {
```

Comment:
Does this function directly have any unit tests ?

Reply:
This part is UI related, can't be tested with JUnit5 tests.